### DISCUSSION TITLE
[UI Feature] Custom sort Liked Tracks and sort by 'most played'

### DISCUSSION THREAD

**Discussion by A-mak88** [KRTirtho/spotube#2040] [Ideas]
Is there an existing issue for this?
 I have searched the existing issues and couldn't find anything.
Category of the Feature
UI Improvements

Credits
Idea inspired from winamp/gmusic

Pain
Liked tracks can only be sorted by newest/oldest added/A-Z,Z-A/Duration/Artist/Album. There are also no further usage details such as amount of times the song has been played.

Suggested Solution
Add a 'custom sort' option which when enabled I can drag + drop songs to the order that best suits me.
Add a played: counter to the left of the song runtime (on android) and allow a 'sort by most played/least played' option. - this is a feature I really miss from google music. Love me some statistics.

Thank you for continual development & support, it is a great app.

**Comment by s-b-repo** [comment]
i wonder if a playlist would be good where you can choose your own song almost like vlc
